### PR TITLE
Separate the "deployed contracts" page into separate pages by network

### PR DESCRIPTION
**Pull request by mymphe**
This PR separates the "deployed contracts" page into separate pages by network. This makes it immediately obvious which network you're working with when checking addresses.

Additionally, I've set up redirects from "/deployed-contracts" to "/deployed-contracts/mainnet" to preserve backlinks.

Preview
image

### CODE REVIEW

File: docs/deployed-contracts/goerli.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- AddRewardProgram: [`0x5560d40b00EA3a64E9431f97B3c79b04e0cdF6F2`](https://goerli.etherscan.io/address/0x5560d40b00EA3a64E9431f97B3c79b04e0cdF6F2)
+- RemoveRewardProgram: [`0x31B68d81125E52fE1aDfe4076F8945D1014753b5`](https://goerli.etherscan.io/address/0x31B68d81125E52fE1aDfe4076F8945D1014753b5)
+- TopUpRewardPrograms: [`0x8180949ac41EF18e844ff8dafE604a195d86Aea9`](https://goerli.etherscan.io/address/0x8180949ac41EF18e844ff8dafE604a195d86Aea9)
+- RewardProgramsRegistry (Referral Program): [`0x4CB0c9987fd670069e4b24c653981E86b261A2ca`](https://goerli.etherscan.io/address/0x4CB0c9987fd670069e4b24c653981E86b261A2ca)
+- AddRewardProgram (Referral Program): [`0xe54ca3e867C52a34d262E94606C7A9371AB820c9`](https://goerli.etherscan.io/address/0xe54ca3e867C52a34d262E94606C7A9371AB820c9)
+- RemoveRewardProgram (Referral Program): [`0x2A0c343087c6cFB721fFa20608A6eD0473C71275`](https://goerli.etherscan.io/address/0x2A0c343087c6cFB721fFa20608A6eD0473C71275)
+- TopUpRewardPrograms (Referral Program): [`0xB1E898faC74c377bEF16712Ba1CD4738606c19Ee`](https://goerli.etherscan.io/address/0xB1E898faC74c377bEF16712Ba1CD4738606c19Ee)
+
+## Görli+Pyrmont testnet
```
Comment (TheDZhon) [inline docs/deployed-contracts/goerli.md:54]: Let's remove Pyrmont setup (the testnet was replaced with Prater in mid 2021)

**Review by TheDZhon — CHANGES_REQUESTED** [review]
It's awesome, thanks! :heart: 

I suggest removing Görli+Pyrmont.

File: docs/deployed-contracts/goerli.md (hunk 1 above)
Comment (kadmil) [reply docs/deployed-contracts/goerli.md:54]: Can do, hadn't used it anywhere for a loooong time

**Review by TheDZhon — APPROVED** [review]
:gem:

**Pull request merged**